js/components/FileChooser.tsx

```diff
@@ -26,7 +26,7 @@ export interface FileChooserProps {
     control?: typeof controlDefault;
 }
 
-const hasPicker: boolean = !!window.showOpenFilePicker;
+const hasPicker = !!window.showOpenFilePicker;
```

Comment:
I had this cast because otherwise the compiler was telling me that it was always `true`. If that's not happening to you, then this is OK.